mylyn.tasks/org.eclipse.mylyn.tasks.core/src/org/eclipse/mylyn/tasks/core/TaskRepository.java

```diff
@@ -547,6 +547,9 @@ public boolean hasCredentials() {
 
 	@Override
 	public int hashCode() {
+		// do not add the URL to the hash
+		// equals use ConnectorKind and URL
+		// org.eclipse.mylyn.tasks.tests.TaskRepositoryManagerTest.testDeletion() fails with the URL
```

Comment:
Useful comment that needs to be promoted to an issue, and then reworked and removed.

Reply:
We now have #261 to track this issue

Reply:
Thank you